crates/biome_js_syntax/src/generated/nodes.rs

```diff
@@ -15,9 +15,7 @@ use biome_rowan::{
     AstNodeList, AstNodeListIterator, AstNodeSlotMap, AstSeparatedList,
     AstSeparatedListNodesIterator,
 };
-#[cfg(feature = "serde")]
```

Comment:
Why this attribute is no longer requuired?

Reply:
1.80.0 got better at catching the features. This one was incorrectly set, and it's not needed. In fact, all the other syntax crates didn't have one.